### PR TITLE
Add Initial Migration

### PR DESCRIPTION
**Pull request by MaximPyatigor**

### CODE REVIEW

File: SAPex/Data/AppDbContext.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SAPex
+{
```
Comment (DoniyorJN) [inline SAPex/Data/AppDbContext.cs:8]: wrong namespace. in your case should be SAPex.Data. but I  think we should extract it into another project

File: SAPex/Startup.cs [hunk 2]
```diff
@@ -13,6 +14,14 @@ namespace SAPex
 {
     public class Startup
     {
+        // string for storing dbsettings
+        private IConfigurationRoot confStringDb;
+
+        public Startup(IWebHostEnvironment hostEnv)
+        {
+            confStringDb = new ConfigurationBuilder().SetBasePath(hostEnv.ContentRootPath).AddJsonFile("dbsettings.json").Build();
```
Comment (DoniyorJN) [inline SAPex/Startup.cs:23]: smth wrong with this we don't need to configure it .
```
public Startup(IConfiguration configuration)
        {
            Configuration = configuration;
        }
  public IConfiguration Configuration { get; }
```
from this conf we can get from appsetting.json any information 
no need to create a new JSON file

File: SAPex/Startup.cs [hunk 3]
```diff
@@ -23,9 +32,11 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-               
             services.AddControllers();
             services.AddSwaggerGen();
+
+            // connection to server, look dbsettings
+            services.AddDbContext<AppDbContext>(options => options.UseSqlServer(confStringDb.GetConnectionString("DefaultConnection")));
         }
```
Comment (DoniyorJN) [inline SAPex/Startup.cs:44]: in IConfiguration we have GetConnectionString, read about this. and rewrite it with GetConnectionString  method

Comment (MaximPyatigor) [reply SAPex/Startup.cs:44]: Done

File: SAPex/Startup.cs (hunk 2 above)
Comment (MaximPyatigor) [reply SAPex/Startup.cs:23]: Done

File: SAPex/Data/AppDbContext.cs (hunk 1 above)
Comment (MaximPyatigor) [reply SAPex/Data/AppDbContext.cs:8]: Done, I had some troubles with git, changes in another pr https://github.com/dnovitsky/exsb21-t4-services/pull/19

**Pull request closed without merging**